src/content/docs/tutorial-infrastructure-monitoring/explore-host-data.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Respond to a crashed app
```

Comment:
this title was jarring to me since I clicked on a link titled `Explorer your host data`